gallery/tutorials/tutorials/ctf.py

```diff
@@ -40,9 +40,9 @@
 from aspire.operators import CTFFilter, RadialCTFFilter
 
 radial_ctf_filter = RadialCTFFilter(
-    pixel_size=1,  # Angstrom
+    pixel_size=1,  # angstrom
     voltage=200,  # kV
-    defocus=10000,  # Angstrom, 10000A = 1um
+    defocus=10000,  # angstrom, 10000A = 1um
```

Comment:
Spaces between value and units (so `10000 A = 1 um`).

Reply:
Great. There's a second instance of this (just below). Can you fix that one as well?